House-of-glass/src/components/Filters/Filters.jsx

```diff
@@ -0,0 +1,134 @@
+    <span className="material-icons">production_quantity_limits</span>
+    <label htmlFor="out-of-stock">In Stock</label>
+    <input 
+    type="checkbox" 
+    name="OUTOFSTOCK" 
+    checked={state.outOfStock===true?true:false} 
+    id="out-of-stock" 
+    onChange={(e)=>dispatch({type:"OUTOFSTOCK", payload:e.target.checked})}/>
+      </div>
+    </div>
+
+<div className='position-relative category-wrapper'>
+  <div className="category-label-container flex-mid-center">
+    <span className="material-icons">category</span>
+    <h6 className='subtitle-1'>Brand</h6>
+  </div>
+
+<div className="pill  align-center gap20 flex-mid-center">
+    <span className="material-icons">production_quantity_limits</span>
+    <label htmlFor="category-1">John Jacobs</label>
+    <input 
+    type="checkbox" 
+    name="CATEGORY" 
+    value="JOHNJACOBS"
+    checked={state.categoryState.find((category)=>category==="JOHNJACOBS")?true:false} 
+    id="category-1" 
+    onChange={(e)=>dispatch({type:"CATEGORY", payload:e.target.value})}/>
+      </div>
+<div className="pill  align-center gap20 flex-mid-center">
+    <span className="material-icons">production_quantity_limits</span>
+    <label htmlFor="category-1">Vogue</label>
+    <input 
+    type="checkbox" 
+    name="CATEGORY" 
+    value="VOGUE"
+    checked={state.categoryState.find((category)=>category==="VOGUE")?true:false} 
+    id="category-1" 
+    onChange={(e)=>dispatch({type:"CATEGORY", payload:e.target.value})}/>
+      </div>
+      
+      <div className="pill  align-center gap20 flex-mid-center">
+    <span className="material-icons">production_quantity_limits</span>
+    <label htmlFor="category-1">Ray Ban</label>
+    <input 
+    type="checkbox" 
+    name="CATEGORY" 
+    value="RAYBAN"
+    checked={state.categoryState.find((category)=>category==="RAYBAN")?true:false} 
+    id="category-1" 
+    onChange={(e)=>dispatch({type:"CATEGORY", payload:e.target.value})}/>
+      </div>
+
+
+      <div className="pill  align-center gap20 flex-mid-center">
+    <span className="material-icons">production_quantity_limits</span>
+    <label htmlFor="category-1">Vincent Chase</label>
+    <input 
+    type="checkbox" 
+    name="CATEGORY" 
+    value="VINCENTCHASE"
+    checked={state.categoryState.find((category)=>category==="VINCENTCHASE")?true:false} 
+    id="category-1" 
+    onChange={(e)=>dispatch({type:"CATEGORY", payload:e.target.value})}/>
+      </div>
+</div>
+
+
+<div className='position-relative category-wrapper'>
+  <div className="category-label-container flex-mid-center">
+    <span className="material-icons">linear_scale</span>
+    <h6 className='subtitle-1'>Range</h6>
+  </div>
+
+  <div className="pill  align-center gap20 flex-mid-center">
+  <input id="price-range"type="range" value={state.minPrice} max={12000} min={1000} step={1000} list="steps" onChange={(e)=>dispatch({type:"PRICERANGE" , payload:e.target.value})}></input>
+      
+      <datalist id="steps">
```

Comment:
Could've been mapped. after extracting it in an array. Makes it future proof

Reply:
Thank You for the suggestion. I will add this in the upcoming PR's.